### ISSUE TITLE
npm install cap fails

### ISSUE THREAD

**Issue by przemyslawpluta** [mscdex/cap#4]
npm

```
npm ERR! cb() never called!
npm ERR! not ok code 0
```

npm log

```
cap@0.0.5 install: `node-gyp rebuild`
npm ERR! `sh "-c" "node-gyp rebuild"` failed with 1
npm ERR!
npm ERR! Failed at the cap@0.0.5 install script.
npm ERR! This is most likely a problem with the cap package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
```

**Comment by hynese** [comment]
On node.js v0.12.2

```
npm update
-
> cap@0.0.12 install /mnt/repos/downloadCat/app/node_modules/cap
> node-gyp rebuild

  CXX(target) Release/obj.target/cap/src/binding.o
../src/binding.cc:336:27: error: expected ';' after expression
        set_immediate_mode(obj->pcap_handle, 1);
                          ^
                          ;
../src/binding.cc:336:33: warning: expression result unused [-Wunused-value]
        set_immediate_mode(obj->pcap_handle, 1);
                           ~~~  ^~~~~~~~~~~
../src/binding.cc:336:46: warning: expression result unused [-Wunused-value]
        set_immediate_mode(obj->pcap_handle, 1);
                                             ^
2 warnings and 1 error generated.
make: *** [Release/obj.target/cap/src/binding.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/ehynes/.nvm/versions/v0.12.2/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Darwin 14.1.0
gyp ERR! command "node" "/Users/ehynes/.nvm/versions/v0.12.2/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /mnt/repos/downloadCat/app/node_modules/cap
gyp ERR! node -v v0.12.2
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok 
npm ERR! Darwin 14.1.0
npm ERR! argv "/Users/ehynes/.nvm/versions/v0.12.2/bin/node" "/Users/ehynes/.nvm/versions/v0.12.2/bin/npm" "update"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4
npm ERR! code ELIFECYCLE

npm ERR! cap@0.0.12 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the cap@0.0.12 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the cap package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls cap
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /mnt/repos/downloadCat/app/npm-debug.log
```

**Comment by mscdex** [comment]
@hynese Please try the master branch and let me know if that works for you.

**Comment by hynese** [comment]
Hi,

Thank you for helping me.

When I did `npm install nan` and then `npm install cap`, it worked.

I'm using io.js v1.2.0. I need to use this specific version for nw.js - https://github.com/nwjs/nw.js)

**Comment by mscdex** [comment]
Changes are now available in v0.0.13 on npm.
